### PR TITLE
Update ms-cloud-germany-transition.md

### PR DESCRIPTION
**Pull request by Ariantor**
Add more in FAQ

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/enterprise/ms-cloud-germany-transition.md [hunk 1]
```diff
@@ -278,6 +278,52 @@ If you have questions, you can contact us or your partner:
...
 - If you are Dynamics 365 Customer Engagement and Power BI customer and also have Office 365, you may submit questions using the &quot;Need Help?&quot; link of the [Microsoft 365 admin center](https://portal.office.de/). Dynamics 365 Customer Engagement support options are located [here](https://docs.microsoft.com/dynamics365/get-started/support/). Power BI support options are located [here](https://powerbi.microsoft.com/support/).
 
+### My customer already has a M365 tenant in the global Microsoft cloud in addition to a Microsoft Cloud Deutschland tenant. Can these two tenants be merged into one as part of the migration?
+
+No, there is no tenant merge capability. Tenants will remain separate and unique as every tenant has its own namespace and unique ID. Microsoft will migrate a Microsoft Cloud Deutschland tenant to the global cloud if desired or else the customer can cancel and abandon it.
+
+
+### What actions are required to be done by most end users as part of the migration?
+The migration is designed to have minimal impact to end users/customers.
+•	Ensure that Office applications are running latest available versions.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/ms-cloud-germany-transition.md:288]:
```suggestion
- Ensure that Office applications are running the latest available versions. 
```

File: microsoft-365/enterprise/ms-cloud-germany-transition.md [hunk 2]
```diff
@@ -278,6 +278,52 @@ If you have questions, you can contact us or your partner:
...
 
+### My customer already has a M365 tenant in the global Microsoft cloud in addition to a Microsoft Cloud Deutschland tenant. Can these two tenants be merged into one as part of the migration?
+
+No, there is no tenant merge capability. Tenants will remain separate and unique as every tenant has its own namespace and unique ID. Microsoft will migrate a Microsoft Cloud Deutschland tenant to the global cloud if desired or else the customer can cancel and abandon it.
+
+
+### What actions are required to be done by most end users as part of the migration?
+The migration is designed to have minimal impact to end users/customers.
+•	Ensure that Office applications are running latest available versions. 
+•	Customers using Skype for Business will transition to Teams as part of the migration and may need to [download and install Teams] (https://docs.microsoft.com/en-us/deployoffice/teams-install) on devices.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/ms-cloud-germany-transition.md:289]:
```suggestion
- Customers using Skype for Business will transition to Teams as part of the migration and may need to [download and install Teams](https://docs.microsoft.com/deployoffice/teams-install) on devices.
```

File: microsoft-365/enterprise/ms-cloud-germany-transition.md [hunk 3]
```diff
@@ -278,6 +278,52 @@ If you have questions, you can contact us or your partner:
...
+•	End users may need to log out of the Office applications and log back in once the migration is complete. 
+•	Customers running the OneDrive Sync client need to log out of their workstation and log in again to allow OneDrive Sync client to log in to the global Azure Active Directory service.
+•	Be aware of new global URLs once migration is complete, notably Outlook Web Access (example: use outlook.office365.com). SharePoint Online clients will continue to successfully connect to the MCD namespace using the existing URL (example: contoso.sharepoint.de).
+
+
+### Which customers are affected by the Azure Active Directory migration? 
+
+All customers of Office365 depend on Azure Active Directory to authenticate and store critical service components needed for operation of Microsoft hosted services.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/ms-cloud-germany-transition.md:297]:
```suggestion
All customers of Office 365 depend on Azure Active Directory to authenticate and store critical service components needed for operation of Microsoft hosted services. 
```

File: microsoft-365/enterprise/ms-cloud-germany-transition.md [hunk 4]
```diff
@@ -278,6 +278,52 @@ If you have questions, you can contact us or your partner:
...
+
+All customers of Office365 depend on Azure Active Directory to authenticate and store critical service components needed for operation of Microsoft hosted services. 
+
+
+### What are the impacts of the Azure Active Directory Migration?
+
+The initial migration of Azure Active Directory in the early phase has no impact to the customer experience. After the final migration stage all services for the customer tenant are fully in the global service. After this final stage the Azure Active Directory service in Microsoft Cloud Deutschland may no longer accept authorization requests or provide access tokens to Office services.
+
+
+### What does it mean to ensure network connectivity to [Office 365 services URLs and IP addresses] (https://aka.ms/o365urls)?
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/ms-cloud-germany-transition.md:305]:
```suggestion
### What does it mean to ensure network connectivity to [Office 365 services, URLs, and IP addresses](https://aka.ms/o365urls)?
```

File: microsoft-365/enterprise/ms-cloud-germany-transition.md [hunk 5]
```diff
@@ -278,6 +278,52 @@ If you have questions, you can contact us or your partner:
...
+
+This article describes the necessary URLs and IP addresses required for proper function of the global service to ensure a good customer experience. In relatively rare cases, some customers attempt to configure network perimeter security in such a way to minimize traffic flows and have restricted access to services to those only as part of the Microsoft Cloud Deutschland service IP ranges.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/ms-cloud-germany-transition.md:307]:
```suggestion
This article describes the necessary URLs and IP addresses required for the proper functioning of the global service to ensure a good customer experience. In relatively rare cases, some customers attempt to configure network perimeter security in such a way to minimize traffic flows and have restricted access to services to those only part of the Microsoft Cloud Deutschland service IP ranges.
```

File: microsoft-365/enterprise/ms-cloud-germany-transition.md [hunk 6]
```diff
@@ -278,6 +278,52 @@ If you have questions, you can contact us or your partner:
...
+
+
+### How do I manage the DNS changes for Exchange Online so mail will continue to flow?
+
+Microsoft-managed IP ranges and DNS zones are transitioned during and as part of the migration to the global service. 
+
+Customer-managed DNS zones such as custom domain MX records are the responsibility of the customer, however, to simplify this migration the customer managed MX record points to an Office 365 service endpoint in the office.de zone and Microsoft manages the migration of this service endpoint automatically.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/ms-cloud-germany-transition.md:314]:
```suggestion
Customer-managed DNS zones such as custom domain MX records are the responsibility of the customer; however, to simplify this migration the customer-managed MX record points to an Office 365 service endpoint in the office.de zone and Microsoft manages the migration of this service endpoint automatically.
```

File: microsoft-365/enterprise/ms-cloud-germany-transition.md [hunk 7]
```diff
@@ -278,6 +278,52 @@ If you have questions, you can contact us or your partner:
...
+
+
+### How do I manage the DNS changes for Skype for Business? 
+ 
+All Skype For Business customers in will transition to Microsoft Teams. The transition of customer Skype DNS zones is not required in the migration to Teams. Customers will be able to sign-into Teams immediately with all functionality after migration.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/ms-cloud-germany-transition.md:319]:
```suggestion
All Skype For Business customers will transition to Microsoft Teams. The transition of customers in Skype DNS zones is not required in the migration to Teams. Customers will be able to sign into Teams immediately with all functionality after migration.
```

File: microsoft-365/enterprise/ms-cloud-germany-transition.md [hunk 8]
```diff
@@ -278,6 +278,52 @@ If you have questions, you can contact us or your partner:
...
+ 
+
+### Will Outlook for iOS and Android work after the migration? 
+
+Yes. Microsoft’s recommendation is all customers run the latest available versions of Office clients including Outlook for iOS and Android clients. Upon completion of the migration to the Office 365 global service, all Office clients will need to log out and log back in to obtain a new Azure Active Directory access token from the global service.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/ms-cloud-germany-transition.md:324]:
```suggestion
Yes. We recommend that all customers run the latest available versions of Office clients including Outlook for iOS and Android clients. Upon completion of the migration to the Office 365 global service, all Office clients will need to log out and log back in to obtain a new Azure Active Directory access token from the global service. 
```

File: microsoft-365/enterprise/ms-cloud-germany-transition.md [hunk 9]
```diff
@@ -278,6 +278,52 @@ If you have questions, you can contact us or your partner:
...
+
+
+### Which customers are affected by the Azure Active Directory migration? 
+
+All customers of Office365 depend on Azure Active Directory to authenticate and store critical service components needed for operation of Microsoft hosted services. 
+
+
+### What are the impacts of the Azure Active Directory Migration?
+
+The initial migration of Azure Active Directory in the early phase has no impact to the customer experience. After the final migration stage all services for the customer tenant are fully in the global service. After this final stage the Azure Active Directory service in Microsoft Cloud Deutschland may no longer accept authorization requests or provide access tokens to Office services.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/ms-cloud-germany-transition.md:302]:
```suggestion
The initial migration of Azure Active Directory in the early phase has no impact on the customer experience. After the final migration stage, all services for the customer tenant are fully in the global service. After this final stage, the Azure Active Directory service in Microsoft Cloud Deutschland may no longer accept authorization requests or provide access tokens to Office services.
```

File: microsoft-365/enterprise/ms-cloud-germany-transition.md [hunk 10]
```diff
@@ -278,6 +278,52 @@ If you have questions, you can contact us or your partner:
...
+
+
+### What actions are required to be done by most end users as part of the migration?
+The migration is designed to have minimal impact to end users/customers.
+•	Ensure that Office applications are running latest available versions. 
+•	Customers using Skype for Business will transition to Teams as part of the migration and may need to [download and install Teams] (https://docs.microsoft.com/en-us/deployoffice/teams-install) on devices.
+•	End users may need to log out of the Office applications and log back in once the migration is complete. 
+•	Customers running the OneDrive Sync client need to log out of their workstation and log in again to allow OneDrive Sync client to log in to the global Azure Active Directory service.
+•	Be aware of new global URLs once migration is complete, notably Outlook Web Access (example: use outlook.office365.com). SharePoint Online clients will continue to successfully connect to the MCD namespace using the existing URL (example: contoso.sharepoint.de).
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/ms-cloud-germany-transition.md:292]:
```suggestion
- Be aware of new global URLs once the migration is complete, notably Outlook Web Access (example: use outlook.office365.com). SharePoint Online clients will continue to successfully connect to the MCD namespace using the existing URL (example: contoso.sharepoint.de).
```

File: microsoft-365/enterprise/ms-cloud-germany-transition.md [hunk 11]
```diff
@@ -278,6 +278,52 @@ If you have questions, you can contact us or your partner:
...
+
+No, there is no tenant merge capability. Tenants will remain separate and unique as every tenant has its own namespace and unique ID. Microsoft will migrate a Microsoft Cloud Deutschland tenant to the global cloud if desired or else the customer can cancel and abandon it.
+
+
+### What actions are required to be done by most end users as part of the migration?
+The migration is designed to have minimal impact to end users/customers.
+•	Ensure that Office applications are running latest available versions. 
+•	Customers using Skype for Business will transition to Teams as part of the migration and may need to [download and install Teams] (https://docs.microsoft.com/en-us/deployoffice/teams-install) on devices.
+•	End users may need to log out of the Office applications and log back in once the migration is complete. 
+•	Customers running the OneDrive Sync client need to log out of their workstation and log in again to allow OneDrive Sync client to log in to the global Azure Active Directory service.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/ms-cloud-germany-transition.md:291]:
```suggestion
- Customers running the OneDrive Sync client need to log out of their workstation and log in again to allow the OneDrive Sync client to log in to the global Azure Active Directory service.
```

File: microsoft-365/enterprise/ms-cloud-germany-transition.md [hunk 12]
```diff
@@ -278,6 +278,52 @@ If you have questions, you can contact us or your partner:
 - For Office 365, you may submit questions using the &quot;Need Help?&quot; link of the [Microsoft 365 admin center](https://portal.office.de/).
 - If you are Dynamics 365 Customer Engagement and Power BI customer and also have Office 365, you may submit questions using the &quot;Need Help?&quot; link of the [Microsoft 365 admin center](https://portal.office.de/). Dynamics 365 Customer Engagement support options are located [here](https://docs.microsoft.com/dynamics365/get-started/support/). Power BI support options are located [here](https://powerbi.microsoft.com/support/).
 
+### My customer already has a M365 tenant in the global Microsoft cloud in addition to a Microsoft Cloud Deutschland tenant. Can these two tenants be merged into one as part of the migration?
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/ms-cloud-germany-transition.md:281]:
```suggestion
### My customer already has a Microsoft 365 tenant in the global Microsoft cloud in addition to a Microsoft Cloud Deutschland tenant. Can these two tenants be merged into one as part of the migration?
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@Ariantor

**Pull request merged**